packages/databricks-vscode/src/vscode-objs/StateStorage.ts

```diff
@@ -58,23 +58,6 @@ const StorageConfigurations = {
         },
     }),
 
-    "databricks.debugging.skipDbConnectInstallForEnvs": withType<string[]>()({
-        location: "global",
-        defaultValue: [],
-        setter: (storage, value) => {
-            if (value === undefined || value.length === 0) {
-                return undefined;
-            }
-            const currentEnvs: string[] = storage.get(
-                "databricks.debugging.skipDbConnectInstallForEnvs"
-            );
-            if (!currentEnvs.includes(value[0])) {
-                currentEnvs.push(value[0]);
-            }
-            return currentEnvs;
-        },
-    }),
```

Comment:
Didn't find it really useful in it's present state, as it was only skipping the "do you want to install" notification, but everything else was still red (e.g. dbconnect button). Was there an intention to silence other things?

Reply:
It was a requirement from the dbconnect team who though it will get too noisy if users really don't want to use dbconnect. I am fine with testing it out without the option, and seeing the response.